sys/compat/linuxkpi/common/include/linux/scatterlist.h

```diff
@@ -119,7 +119,7 @@ sg_assign_page(struct scatterlist *sg, struct page *page)
 {
 	unsigned long page_link = sg->page_link & SG_PAGE_LINK_MASK;
 
-	sg->page_link = page_link | (unsigned long)page;
```

Comment:
I would suggest splitting out the changes here for sg_* into a separate commit ("linuxkpi scatterlist: Use uintptr_t for pointers") or some such.

Reply:
Splitted out.